README.md

```diff
@@ -24,6 +24,10 @@ __When you are done, create a production-ready version of the JS bundle:__
 
 ```npm run build```
 
+## Deployment
+
+Changes to master are deployed to S3 via [Travis-CI](https://travis-ci.org/zooniverse/vox).
+
```

Comment:
It might be helpful to add this as well?
"If you're adding new deps, you'll need to clear the cache before rebuilding."